R/parser-intelliflex.R

```diff
@@ -5,10 +5,10 @@ require(dplyr)
 #' @param path Path to the INTELLIFLEX file
 #'
 #' @import dplyr
-#'
-#' @export
+#' @import utils
+#' @import stringr
 read_intelliflex_format <- function(path) {
-  df <- read.csv(path)
+  df <- utils::read.csv(path, stringsAsFactors = FALSE)
```

Comment:
same here

Reply:
I thought that the user will interact with our program mainly using the `read_luminex` function. Functions like `read_intelliflex_format` or `read_layout_data` output data that is not integrated into the plate object, so there is no reason to use it. Maybe in the case of the `layout_matrix` that might be useful sometimes, but it also might confuse the user with function they should use